src/api/FactsController.js

```diff
@@ -1,4 +1,4 @@
-export async function retrieveAnimalFact() {
+export default async function retrieveAnimalFact() {
```

Comment:
it is confusing when exported default has a different name from filename. Use export instead